test/ts-scenario/config/handlers/sample-transaction-event-handler.ts

```diff
@@ -38,23 +29,14 @@ class SampleTransactionEventHandler {
 	 * @param {Object} [options] Additional configuration options.
 	 * @param {Number} [options.commitTimeout] Time in seconds to wait for commit events to be received.
 	 */
-	constructor(transaction: Transaction, eventHubs: ChannelEventHub[], options: object) {
-		this.transactionId = transaction.getTransactionID().getTransactionID();
-		this.eventHubs = eventHubs;
-
-		const defaultOptions: any = {
-			commitTimeout: 120 // 2 minutes (120 seconds)
-		};
-		this.options = Object.assign(defaultOptions, options);
+	constructor(transaction: Transaction, options: any) {
```

Comment:
This needs to be reverted to a real working usage example. It is exposed as an example in the docs and also confirms that it is possible for end users to implement custom handlers using only the public API.